source/features/parse-backticks.tsx

```diff
@@ -41,6 +43,40 @@ function init(): void {
 			parseBackticks(element);
 		}
 	});
+
+	// `isRepoSearch`
+	observe('#issue_search_results .f4:not(.rgh-backticks-already-parsed)', {
+		add(element) {
+			const child = element.firstElementChild!;
+			const keywords = [...child.querySelectorAll('em')].map(element => element.textContent);
+			// Combine text content to enable backticks parsing
+			child.textContent = `${child.textContent!}`;
```

Comment:
Can't just do `child.textContent = child.textContent` because the linter like it go complicated